.github/workflows/integration.yml

```
@@ -0,0 +1,65 @@
+name: integration-test
+
+on:
+  push:
+    branches:
+      - master
+      # - <your-branch>    # put your branch name here to test it @ GH Actions
+  pull_request:
+    branches:
+      - master
```

"On pull request" action for `yajsapi` unit tests looks like this:
```suggestion
      - master
      - b0.*
```

---

added in https://github.com/golemfactory/yapapi/pull/281/commits/0d80b77dcf376c1adc98da244f60945d4daf27c7